entrypoint.sh

```diff
@@ -1,9 +1,17 @@
 #!/bin/bash
 
+export TARGET_API="https://api.qa.notprod.lev.homeoffice.gov.uk"
+
 if [ -z "$1" ] ; then
   echo "Error: expected to be called with a configuration file"
   exit 1
 fi
+if ! artillery -V &> /dev/null ; then
+  echo "Error: Artillery is not installed, either run:
+  - npm i # to install it locally, or
+  - npm install -g artillery # for global installation"
+  exit 1
+fi
```

Comment:
Does this not require that it be install globally?
Is it worth adding `./node_modules/.bin` to the path?

Reply:
Yes, good point, sometimes I forget that I already have this as part of my `.bashrc`. Refactored `./entrypoint.sh`